botfront/imports/ui/components/stories/common/UserUtterancesContainer.jsx

```diff
@@ -0,0 +1,96 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+
+import IconButton from '../../common/IconButton';
+import UserUtteranceContainer from './UserUtteranceContainer';
+import { NEW_INTENT } from '../../../../lib/story_controller';
+
+const UserUtterancesContainer = (props) => {
+    const {
+        deletable, value, onChange, onDelete,
+    } = props;
+
+    const somethingIsBeingInput = useMemo(() => value.some(d => d === null), [value]);
+
```

Comment:
Minor: `value.some(d => d === null), [value])`; may be use a more explicit variable name ? d is not very clear